### PR TITLE
Adding build script for npm package - jsbn. Default version 'master'.

### PR DESCRIPTION
**Pull request by gitssinghal**
Added script and license file.

### CODE REVIEW

File: j/jsbn/jsbn_ubi_8.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
+
+PACKAGE_NAME=jsbn
+PACKAGE_URL=https://github.com/andyperlitch/jsbn
+
+yum -y update
+yum -y install git wget gcc-c++ make python2
+
+wget https://nodejs.org/dist/latest-v12.x/node-v12.22.3-linux-ppc64le.tar.gz
```
Comment (seth-priya) [inline j/jsbn/jsbn_ubi_8.sh:30]: Can we install nod using dnf or nvm, that would be cleaner

File: r/rd3_ubi_8.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+yum -y update
+yum -y install git wget gcc-c++ make python2 bzip2 fontconfig-devel
+
+tar -xvf phantomjs-2.1.1-linux-ppc64.tar.bz2
+ln -s /phantomjs-2.1.1-linux-ppc64/bin/phantomjs /usr/local/bin/phantomjs
+export PATH=$PATH:/phantomjs-2.1.1-linux-ppc64/bin/phantomjs
+
+#gulp error with node 12, hence using node 10
+wget https://nodejs.org/dist/v10.9.0/node-v10.9.0-linux-ppc64le.tar.gz
```
Comment (seth-priya) [inline r/rd3_ubi_8.sh:35]: Better way to install node would be using "nvm"

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: j/jsbn/jsbn_ubi_8.sh (hunk 1 above)
Comment (gitssinghal) [reply j/jsbn/jsbn_ubi_8.sh]: installed through nvm. made changes

File: r/rd3_ubi_8.sh (hunk 2 above)
Comment (gitssinghal) [reply r/rd3_ubi_8.sh]: made the changes

**Review by seth-priya — APPROVED** [review]

**Pull request merged**